agent/lib_aws_sdk_php.h

```diff
@@ -74,6 +74,23 @@ extern void nr_lib_aws_sdk_php_sqs_handle(nr_segment_t* segment,
                                           size_t command_name_len,
                                           NR_EXECUTE_PROTO);
 
+/*
+ * Purpose : Handle when a LambdaClient::invoke command happens
+ *
+ * Params  : 1. NR_EXECUTE_ORIG_ARGS (execute_data, func_return_value)
+ *           2. cloud_attrs : the cloud attributes pointer to be
+ *              populated with the ARN
+ *
+ * Returns :
+ *
+ */
+void nr_aws_lambda_invoke(NR_EXECUTE_PROTO, nr_segment_cloud_attrs_t* cloud_attrs);
```

Comment:
The function name suggests that calling this function will invoke aws lambda where in fact this is a helper function that parses arguments passed to `Aws\Lambda\LambdaClient::invoke`. The comment right above where this function is used, `/* reconstruct the ARN */`, tells more what this function does than its name. Please adjust the function name to better match its purpose. Maybe `nr_aws_sdk_lambda_client_invoke_parse_args`?

Reply:
`nr_aws_lambda_invoke` is very similar in function to `nr_lib_aws_sdk_php_sqs_parse_queueurl` - both parse stuff (arguments in case of `nr_aws_lambda_invoke` and queue url in case of `nr_lib_aws_sdk_php_sqs_parse_queueurl`).

Reply:
changed in [5c45a0e](https://github.com/newrelic/newrelic-php-agent/pull/1023/commits/5c45a0ef346dbb2bf7e1b28a8ed0de4cbabcbb3f)